diff --git a/node_modules/@coveo/semantic-monorepo-tools/dist/git/getLastGitTag.d.ts b/node_modules/@coveo/semantic-monorepo-tools/dist/git/getLastGitTag.d.ts
index 9c99911..85e65ec 100644
--- a/node_modules/@coveo/semantic-monorepo-tools/dist/git/getLastGitTag.d.ts
+++ b/node_modules/@coveo/semantic-monorepo-tools/dist/git/getLastGitTag.d.ts
@@ -3,4 +3,7 @@
  * @param prefix only consider tag that has this prefix. Will return the last tag of the current branch by default.
  * @return an array containing the last tag and the process that was spawned
  */
-export default function (prefix?: string): Promise<string>;
+export default function ({
+    prefix,
+    onBranch,
+  }: { prefix?: string; onBranch?: string } = {}): Promise<string>;
diff --git a/node_modules/@coveo/semantic-monorepo-tools/dist/git/getLastGitTag.js b/node_modules/@coveo/semantic-monorepo-tools/dist/git/getLastGitTag.js
index f0ecdcf..1cfe9b1 100644
--- a/node_modules/@coveo/semantic-monorepo-tools/dist/git/getLastGitTag.js
+++ b/node_modules/@coveo/semantic-monorepo-tools/dist/git/getLastGitTag.js
@@ -5,11 +5,14 @@ import gitLogger from "./utils/gitLogger.js";
  * @param prefix only consider tag that has this prefix. Will return the last tag of the current branch by default.
  * @return an array containing the last tag and the process that was spawned
  */
-export default async function (prefix) {
+export default async function ({prefix, onBranch}) {
     const gitParams = ["describe", "--tags", "--abbrev=0"];
     if (prefix) {
         gitParams.push(`--match=${prefix}*`);
     }
+    if(onBranch){
+        gitParams.push(onBranch);
+    }
     const gitPs = await spawn("git", gitParams, gitLogger);
     return gitPs.stdout.trim();
 }
