diff --git a/node_modules/@coveo/semantic-monorepo-tools/dist/git/getCommits.js b/node_modules/@coveo/semantic-monorepo-tools/dist/git/getCommits.js
index 302c25a..1bbd782 100644
--- a/node_modules/@coveo/semantic-monorepo-tools/dist/git/getCommits.js
+++ b/node_modules/@coveo/semantic-monorepo-tools/dist/git/getCommits.js
@@ -13,7 +13,7 @@ export default async function (projectPath, from, to = "HEAD") {
     const delimiter = `<--- ${randomBytes(64).toString("hex")} --->`;
     const gitParams = [
         "log",
-        `--pretty="format:%B%n-hash-%n%H ${delimiter}"`,
+        `--format=%B%n-hash-%n%H%n${delimiter}`,
         "--dense",
         `${from}..${to}`,
     ].concat(getOptionalPositionalArgument(projectPath));
