diff --git a/node_modules/@stencil/core/compiler/stencil.js b/node_modules/@stencil/core/compiler/stencil.js
index 05c98c9..1ae35e5 100644
--- a/node_modules/@stencil/core/compiler/stencil.js
+++ b/node_modules/@stencil/core/compiler/stencil.js
@@ -234904,8 +234904,8 @@ var getTsOptionsToExtend = (config) => {
     // if the `DIST_TYPES` output target is present then we'd like to emit
     // declaration files
     declaration: config.outputTargets.some(isOutputTargetDistTypes),
-    module: import_typescript.default.ModuleKind.ESNext,
-    moduleResolution: import_typescript.default.ModuleResolutionKind.NodeJs,
+    module: import_typescript.default.ModuleKind.ES2022,
+    moduleResolution: import_typescript.default.ModuleResolutionKind.Bundler,
     noEmitOnError: false,
     outDir: config.cacheDir || config.sys.tmpDirSync(),
     sourceMap: config.sourceMap,
@@ -259592,7 +259592,7 @@ var visitClassDeclaration = (config, diagnostics, typeChecker, program, classNod
       importAliasMap.get("AttachInternals")
     );
   }
-  const updatedClassFields = handleClassFields(classNode, filteredMethodsAndFields, typeChecker, importAliasMap);
+  const updatedClassFields = handleClassFields(classNode, decoratedMembers, typeChecker, importAliasMap, filteredMethodsAndFields);
   validateMethods(diagnostics, classMembers);
   const currentDecorators = retrieveTsDecorators(classNode);
   return import_typescript54.default.factory.updateClassDeclaration(
@@ -259632,9 +259632,6 @@ var removeStencilMethodDecorators = (classMembers, diagnostics, importAliasMap)
           member2.body
         );
       } else if (import_typescript54.default.isPropertyDeclaration(member2)) {
-        if (shouldInitializeInConstructor(member2, importAliasMap)) {
-          return member2;
-        } else {
           const modifiers = retrieveTsModifiers(member2);
           return import_typescript54.default.factory.updatePropertyDeclaration(
             member2,
@@ -259644,7 +259641,6 @@ var removeStencilMethodDecorators = (classMembers, diagnostics, importAliasMap)
             member2.type,
             member2.initializer
           );
-        }
       } else {
         const err2 = buildError(diagnostics);
         err2.messageText = "Unknown class member encountered!";
@@ -259668,10 +259664,9 @@ var filterDecorators = (decorators, excludeList) => {
   }
   return decorators;
 };
-function handleClassFields(classNode, classMembers, typeChecker, importAliasMap) {
+function handleClassFields(classNode, classMembers, typeChecker, importAliasMap, updatedClassMembers) {
   var _a;
   const statements = [];
-  const updatedClassMembers = [];
   for (const member2 of classMembers) {
     if (shouldInitializeInConstructor(member2, importAliasMap) && import_typescript54.default.isPropertyDeclaration(member2)) {
       const memberName = tsPropDeclNameAsString(member2, typeChecker);
@@ -259686,8 +259681,6 @@ function handleClassFields(classNode, classMembers, typeChecker, importAliasMap)
           )
         )
       );
-    } else {
-      updatedClassMembers.push(member2);
     }
   }
   if (statements.length === 0) {
