diff --git a/node_modules/@stencil/core/compiler/stencil.js b/node_modules/@stencil/core/compiler/stencil.js
index d4d0311..692d9a8 100644
--- a/node_modules/@stencil/core/compiler/stencil.js
+++ b/node_modules/@stencil/core/compiler/stencil.js
@@ -256504,6 +256504,7 @@ const getTranspileConfig = (input) => {
     target: getTranspileConfigOpt(input.target, VALID_TARGET, 'latest'),
   };
   const tsCompilerOptions = {
+    experimentalDecorators: true,
     // best we always set this to true
     allowSyntheticDefaultImports: true,
     // best we always set this to true
