diff --git a/node_modules/@stencil/core/compiler/stencil.js b/node_modules/@stencil/core/compiler/stencil.js
index 4f3a6e3..487f815 100644
--- a/node_modules/@stencil/core/compiler/stencil.js
+++ b/node_modules/@stencil/core/compiler/stencil.js
@@ -238083,8 +238083,8 @@ var getTsOptionsToExtend = (config) => {
     // if the `DIST_TYPES` output target is present then we'd like to emit
     // declaration files
     declaration: config.outputTargets.some(isOutputTargetDistTypes),
-    module: import_typescript.default.ModuleKind.ESNext,
-    moduleResolution: import_typescript.default.ModuleResolutionKind.NodeJs,
+    module: import_typescript.default.ModuleKind.ES2022,
+    moduleResolution: import_typescript.default.ModuleResolutionKind.Bundler,
     noEmitOnError: false,
     outDir: config.cacheDir || config.sys.tmpDirSync(),
     sourceMap: config.sourceMap,
@@ -262850,7 +262850,7 @@ var visitClassDeclaration = (config, diagnostics, typeChecker, program, classNod
       importAliasMap.get("AttachInternals")
     );
   }
-  const updatedClassFields = handleClassFields(classNode, filteredMethodsAndFields, typeChecker, importAliasMap);
+  const updatedClassFields = handleClassFields(classNode, decoratedMembers, typeChecker, importAliasMap, filteredMethodsAndFields);
   validateMethods(diagnostics, classMembers);
   const currentDecorators = retrieveTsDecorators(classNode);
   return import_typescript54.default.factory.updateClassDeclaration(
@@ -262890,9 +262890,6 @@ var removeStencilMethodDecorators = (classMembers, diagnostics, importAliasMap)
           member2.body
         );
       } else if (import_typescript54.default.isPropertyDeclaration(member2)) {
-        if (shouldInitializeInConstructor(member2, importAliasMap)) {
-          return member2;
-        } else {
           const modifiers = retrieveTsModifiers(member2);
           return import_typescript54.default.factory.updatePropertyDeclaration(
             member2,
@@ -262926,10 +262923,9 @@ var filterDecorators = (decorators, excludeList) => {
   }
   return decorators;
 };
-function handleClassFields(classNode, classMembers, typeChecker, importAliasMap) {
+function handleClassFields(classNode, classMembers, typeChecker, importAliasMap, updatedClassMembers) {
   var _a;
   const statements = [];
-  const updatedClassMembers = [];
   for (const member2 of classMembers) {
     if (shouldInitializeInConstructor(member2, importAliasMap) && import_typescript54.default.isPropertyDeclaration(member2)) {
       const memberName = tsPropDeclNameAsString(member2, typeChecker);
@@ -262944,8 +262940,6 @@ function handleClassFields(classNode, classMembers, typeChecker, importAliasMap)
           )
         )
       );
-    } else {
-      updatedClassMembers.push(member2);
     }
   }
   if (statements.length === 0) {
