diff --git a/dist/index.d.ts b/dist/index.d.ts
index f22a4c4..626c85e 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -7,6 +7,8 @@ type StorybookHelpersOptions = {
     hideArgRef?: boolean;
     /** sets the custom type reference in the Custom Elements Manifest */
     typeRef?: string;
+    /** Selector for the element containing the rendered story */
+    containerSelector?: string;
     /** Adds a <script> tag where a `component` variable will reference the story's component */
     setComponentVariable?: boolean;
     /** renders default values for attributes and CSS properties */
@@ -23,6 +25,8 @@ type Options = StorybookHelpersOptions;
 type StoryOptions = {
     /** Categories to exclude from these stories */
     excludeCategories?: Array<Categories>;
+    /** Selector for the element containing the rendered story */
+    containerSelector?: string;
     /** Adds a <script> tag where a `component` variable will reference the story's component */
     setComponentVariable?: boolean;
 };
diff --git a/dist/index.js b/dist/index.js
index cc73452..10f7199 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -629,7 +629,7 @@ function transformToScoped(template, tagSelector) {
   ${styleContent}
 }`;
 }
-function getTemplate(component, args, slot, argTypes, excludeCategories, setComponentVariable) {
+function getTemplate(component, args, slot, argTypes, excludeCategories, setComponentVariable, containerSelector) {
   if (!args) {
     return html`<${unsafeStatic(component.tagName)}></${unsafeStatic(component.tagName)}>`;
   }
@@ -654,11 +654,23 @@ function getTemplate(component, args, slot, argTypes, excludeCategories, setComp
     excludeCategories,
     scopeSelector
   );
-  syncControls(component);
+  syncControls(component, containerSelector);
   return html`${styleTemplate}
 <${unsafeStatic(component.tagName)} ${spread(operators)}>${slotsTemplate}${slot || ""}</${unsafeStatic(component.tagName)}>
 ${options2.setComponentVariable || setComponentVariable ? unsafeHTML(
-    "<script>\n  window.component = document.querySelector('" + component.tagName + "');\n</script>"
+    `<script>
+      var containerElement = ${containerSelector ? `document.querySelector('${containerSelector}')` : 'document'};
+      if (!containerElement) {
+        containerElement = document;
+      }
+      if (containerElement instanceof HTMLTemplateElement) {
+        containerElement = containerElement.content;
+      }
+      if (containerElement.shadowRoot) {
+        containerElement = containerElement.shadowRoot;
+      }
+      window.component = containerElement.querySelector('` + component.tagName + `');
+    }\n</script>`
   ) : ""}
 `;
 }
@@ -806,15 +818,27 @@ function getSlotsTemplate(component, args, excludeCategories) {
 ${slotTemplates}
 `) : "";
 }
-function syncControls(component) {
+function syncControls(component, containerSelector) {
   setArgObserver(component);
   setTimeout(() => {
-    const selectedComponent = document.querySelector(component.tagName);
-    if (selectedComponent && argObserver) {
-      argObserver.observe(selectedComponent, {
-        attributes: true
-      });
+    let containerElement = containerSelector ? document.querySelector(containerSelector) : document;
+    if (!containerElement) {
+      containerElement = document;
+    }
+    if (containerElement instanceof HTMLTemplateElement) {
+      containerElement = containerElement.content;
+    }
+    if (containerElement.shadowRoot) {
+      containerElement = containerElement.shadowRoot;
+    }
+    const selectedComponent = containerElement.querySelector(component.tagName);
+    if (!selectedComponent) {
+      return;
     }
+    argObserver?.observe(selectedComponent, {
+      attributes: true,
+      attributeOldValue: true,
+    });
   });
 }
 function setArgObserver(component) {
@@ -822,7 +846,7 @@ function setArgObserver(component) {
     return;
   }
   let isUpdating = false;
-  const updateArgs = useArgs()[1];
+  const [args, updateArgs] = useArgs();
   const { attrArgs: attributes } = getAttributesAndProperties(component);
   if (argObserver) {
     return;
@@ -830,17 +854,27 @@ function setArgObserver(component) {
   argObserver = new MutationObserver((mutations) => {
     mutations.forEach((mutation) => {
       if (isUpdating) return;
-      const attribute = attributes[`${mutation.attributeName}`];
-      if (!attribute) return;
       if (mutation.attributeName === "class" && isUpdating) {
         return;
       }
-      isUpdating = true;
+      const attribute = attributes[`${mutation.attributeName}`];
+      if (!attribute) { return; }
+      if (mutation.oldValue === mutation.target.getAttribute(mutation.attributeName || "")) {
+        return;
+      }
       if (attribute?.control === "boolean" || attribute?.control?.type === "boolean") {
+        if (args[`${mutation.attributeName}`] === mutation.target.hasAttribute(mutation.attributeName || "")) {
+          return;
+        }
+        isUpdating = true;
         updateArgs({
           [`${mutation.attributeName}`]: mutation.target?.hasAttribute(mutation.attributeName || "")
         });
       } else {
+        if (args[`${mutation.attributeName}`] == mutation.target.getAttribute(mutation.attributeName || "")) {
+          return;
+        }
+        isUpdating = true;
         updateArgs({
           [`${mutation.attributeName}`]: mutation.target.getAttribute(mutation.attributeName || "")
         });
@@ -897,7 +931,8 @@ function getStorybookHelpers(tagName, options3) {
       slot,
       argTypes,
       options3?.excludeCategories || [],
-      options3?.setComponentVariable
+      options3?.setComponentVariable,
+      options3?.containerSelector
     )
   };
   return helpers;
