diff --git a/esm/index.js b/esm/index.js
index a82986b88f0392243dbbb84dcfd8d3161b170043..fce6b4d89921c36ef24b2930079d4ce979e91400 100644
--- a/esm/index.js
+++ b/esm/index.js
@@ -1,6 +1,6 @@
-import * as C from './constant';
-import en from './locale/en';
-import U from './utils';
+import * as C from './constant.js';
+import en from './locale/en.js';
+import U from './utils.js';
 var L = 'en'; // global locale
 
 var Ls = {}; // global loaded locale
diff --git a/esm/locale/af.js b/esm/locale/af.js
index ce0c2855f842e4d53df4bf256f2dc24cb6814317..dad2b7ec28e688940ba16bff6973e7149f5b0c1f 100644
--- a/esm/locale/af.js
+++ b/esm/locale/af.js
@@ -1,5 +1,5 @@
 // Afrikaans [af]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'af',
   weekdays: 'Sondag_Maandag_Dinsdag_Woensdag_Donderdag_Vrydag_Saterdag'.split('_'),
diff --git a/esm/locale/am.js b/esm/locale/am.js
index cf25510c7ba068832c36bed789c79f21f04cae21..f51cf842fad1872c56c895a75899589806786575 100644
--- a/esm/locale/am.js
+++ b/esm/locale/am.js
@@ -1,5 +1,5 @@
 // Amharic [am]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'am',
   weekdays: 'እሑድ_ሰኞ_ማክሰኞ_ረቡዕ_ሐሙስ_አርብ_ቅዳሜ'.split('_'),
diff --git a/esm/locale/ar-dz.js b/esm/locale/ar-dz.js
index 3ecc04fca1308f0ca285f6869ffcf33a7c188910..8ddf777c7bde28b7fe3b3a483a73355eb6565aa6 100644
--- a/esm/locale/ar-dz.js
+++ b/esm/locale/ar-dz.js
@@ -1,5 +1,5 @@
 // Arabic (Algeria) [ar-dz]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ar-dz',
   weekdays: 'الأحد_الإثنين_الثلاثاء_الأربعاء_الخميس_الجمعة_السبت'.split('_'),
diff --git a/esm/locale/ar-iq.js b/esm/locale/ar-iq.js
index dfe31bf222b3e68338d886b6786deaff3721efb8..55e11e6e8c686ae9cc9474196f41146bf06548fe 100644
--- a/esm/locale/ar-iq.js
+++ b/esm/locale/ar-iq.js
@@ -1,5 +1,5 @@
 //  Arabic (Iraq) [ar-iq]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ar-iq',
   weekdays: 'الأحد_الإثنين_الثلاثاء_الأربعاء_الخميس_الجمعة_السبت'.split('_'),
diff --git a/esm/locale/ar-kw.js b/esm/locale/ar-kw.js
index 73bf90a7a8bf2ed9266e3fff3ca997f0f4fcf645..e2f88d178f633bf23de22a5a49c8c7e278c1548d 100644
--- a/esm/locale/ar-kw.js
+++ b/esm/locale/ar-kw.js
@@ -1,5 +1,5 @@
 // Arabic (Kuwait) [ar-kw]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ar-kw',
   weekdays: 'الأحد_الإثنين_الثلاثاء_الأربعاء_الخميس_الجمعة_السبت'.split('_'),
diff --git a/esm/locale/ar-ly.js b/esm/locale/ar-ly.js
index 5caa8698046ad949c2b581afdb1ac65af37d7599..6a5f45aa4d882eef28d0935d68906ba6fe2849b2 100644
--- a/esm/locale/ar-ly.js
+++ b/esm/locale/ar-ly.js
@@ -1,5 +1,5 @@
 // Arabic (Lybia) [ar-ly]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ar-ly',
   weekdays: 'الأحد_الإثنين_الثلاثاء_الأربعاء_الخميس_الجمعة_السبت'.split('_'),
diff --git a/esm/locale/ar-ma.js b/esm/locale/ar-ma.js
index ed6dfef07f755dcd96dc0b91d583df7356e2545b..d3547f7b49fa17474c5f6999e8b3cf0af557d72d 100644
--- a/esm/locale/ar-ma.js
+++ b/esm/locale/ar-ma.js
@@ -1,5 +1,5 @@
 // Arabic (Morocco) [ar-ma]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ar-ma',
   weekdays: 'الأحد_الإثنين_الثلاثاء_الأربعاء_الخميس_الجمعة_السبت'.split('_'),
diff --git a/esm/locale/ar-sa.js b/esm/locale/ar-sa.js
index 8eb9687e1868fe3e49d979a9d2cc21c42dbcbe8f..cfdb6231a924221b8f1e68072cc1c709c59c0b8e 100644
--- a/esm/locale/ar-sa.js
+++ b/esm/locale/ar-sa.js
@@ -1,5 +1,5 @@
 // Arabic (Saudi Arabia) [ar-sa]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ar-sa',
   weekdays: 'الأحد_الإثنين_الثلاثاء_الأربعاء_الخميس_الجمعة_السبت'.split('_'),
diff --git a/esm/locale/ar-tn.js b/esm/locale/ar-tn.js
index 3c1f2b01d1a2ba360277be256182c69adfefcf89..ab5c27cb1a3d830842f54d157b86fb0451bbfbd9 100644
--- a/esm/locale/ar-tn.js
+++ b/esm/locale/ar-tn.js
@@ -1,5 +1,5 @@
 //  Arabic (Tunisia) [ar-tn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ar-tn',
   weekdays: 'الأحد_الإثنين_الثلاثاء_الأربعاء_الخميس_الجمعة_السبت'.split('_'),
diff --git a/esm/locale/ar.js b/esm/locale/ar.js
index 78b99b8bf6f6bd96ae517d5b837009f9d96010d2..99dc3f4f83e1f714228b5401de108409ca9a4fcd 100644
--- a/esm/locale/ar.js
+++ b/esm/locale/ar.js
@@ -1,5 +1,5 @@
 // Arabic [ar]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var months = 'يناير_فبراير_مارس_أبريل_مايو_يونيو_يوليو_أغسطس_سبتمبر_أكتوبر_نوفمبر_ديسمبر'.split('_');
 var symbolMap = {
   1: '١',
diff --git a/esm/locale/az.js b/esm/locale/az.js
index 3505c8a752d59a86f1562a541212fe8319f44e16..4dcc8a706f0fdd4d2d47e188a17e98d9f3a409d8 100644
--- a/esm/locale/az.js
+++ b/esm/locale/az.js
@@ -1,5 +1,5 @@
 // Azerbaijani [az]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'az',
   weekdays: 'Bazar_Bazar ertəsi_Çərşənbə axşamı_Çərşənbə_Cümə axşamı_Cümə_Şənbə'.split('_'),
diff --git a/esm/locale/be.js b/esm/locale/be.js
index 5642e397ffe61459f5731310ae2fbc77dd7c5f97..3e9a14f152ceb3929831327030c9e562c4fb5d85 100644
--- a/esm/locale/be.js
+++ b/esm/locale/be.js
@@ -1,5 +1,5 @@
 // Belarusian [be]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'be',
   weekdays: 'нядзелю_панядзелак_аўторак_сераду_чацвер_пятніцу_суботу'.split('_'),
diff --git a/esm/locale/bg.js b/esm/locale/bg.js
index e4e2a34ad998fedd4191506ac162048807871868..3e2ca76344c1b9a0adfc8bf646c9257432c6dc2f 100644
--- a/esm/locale/bg.js
+++ b/esm/locale/bg.js
@@ -1,5 +1,5 @@
 // Bulgarian [bg]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'bg',
   weekdays: 'неделя_понеделник_вторник_сряда_четвъртък_петък_събота'.split('_'),
diff --git a/esm/locale/bi.js b/esm/locale/bi.js
index 6230f251fb3cacf7e8b02a220c67ec6261d5fcc8..09d53525e66cd7ea207ada94f4cfabe023ec6739 100644
--- a/esm/locale/bi.js
+++ b/esm/locale/bi.js
@@ -1,5 +1,5 @@
 // Bislama [bi]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'bi',
   weekdays: 'Sande_Mande_Tusde_Wenesde_Tosde_Fraede_Sarade'.split('_'),
diff --git a/esm/locale/bm.js b/esm/locale/bm.js
index 0d610930985c048c32cad0c77b3fd5073e5d6ee3..4dc6bc090d2dcedbcb42d41dce45bbc698a8776e 100644
--- a/esm/locale/bm.js
+++ b/esm/locale/bm.js
@@ -1,5 +1,5 @@
 // Bambara [bm]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'bm',
   weekdays: 'Kari_Ntɛnɛn_Tarata_Araba_Alamisa_Juma_Sibiri'.split('_'),
diff --git a/esm/locale/bn-bd.js b/esm/locale/bn-bd.js
index f13b660ba4ddd235e97ac0e38566df2a0a003f80..c1906abdc3b5594616734bb581904f64f49c2442 100644
--- a/esm/locale/bn-bd.js
+++ b/esm/locale/bn-bd.js
@@ -1,5 +1,5 @@
 // Bengali (Bangladesh) [bn-bd]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var symbolMap = {
   1: '১',
   2: '২',
diff --git a/esm/locale/bn.js b/esm/locale/bn.js
index 25fc1703509cd5455370e17abbdadf64172976b3..f2fe7d46fea15f1831779351b45c35f4294c705f 100644
--- a/esm/locale/bn.js
+++ b/esm/locale/bn.js
@@ -1,5 +1,5 @@
 // Bengali [bn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var symbolMap = {
   1: '১',
   2: '২',
diff --git a/esm/locale/bo.js b/esm/locale/bo.js
index fce33446057510f05bcdebeff39bfb3c8ea7fb10..e3731e4a70d7116cfacfa823e8476b90ba114340 100644
--- a/esm/locale/bo.js
+++ b/esm/locale/bo.js
@@ -1,5 +1,5 @@
 // Tibetan [bo]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'bo',
   weekdays: 'གཟའ་ཉི་མ་_གཟའ་ཟླ་བ་_གཟའ་མིག་དམར་_གཟའ་ལྷག་པ་_གཟའ་ཕུར་བུ_གཟའ་པ་སངས་_གཟའ་སྤེན་པ་'.split('_'),
diff --git a/esm/locale/br.js b/esm/locale/br.js
index d18b4fea5d6ec6137da8c2a2e89b3d994d66ad89..eb770e31e2a646926a61bea1c3be77c3a60ea315 100644
--- a/esm/locale/br.js
+++ b/esm/locale/br.js
@@ -1,5 +1,5 @@
 // Breton [br]
-import dayjs from '../index';
+import dayjs from '../index.js';
 
 function lastNumber(number) {
   if (number > 9) {
diff --git a/esm/locale/bs.js b/esm/locale/bs.js
index 328a1fe38bd6e19c60608cf1d6c36da001e948f9..68603e283c55161d059dc8fc64fb493f7011c97f 100644
--- a/esm/locale/bs.js
+++ b/esm/locale/bs.js
@@ -1,5 +1,5 @@
 // Bosnian [bs]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'bs',
   weekdays: 'nedjelja_ponedjeljak_utorak_srijeda_četvrtak_petak_subota'.split('_'),
diff --git a/esm/locale/ca.js b/esm/locale/ca.js
index 94fc0b9c69172ef65ca2ad6aa81748ff9931a148..33428790686dc799f06351ffd09b90e955df3283 100644
--- a/esm/locale/ca.js
+++ b/esm/locale/ca.js
@@ -1,5 +1,5 @@
 // Catalan [ca]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ca',
   weekdays: 'Diumenge_Dilluns_Dimarts_Dimecres_Dijous_Divendres_Dissabte'.split('_'),
diff --git a/esm/locale/cs.js b/esm/locale/cs.js
index 165b662c6bc1f74d8d2edb4edeafd07e9b8fcf0b..76c4735cca92ce534b73cce9396a542fdde53fec 100644
--- a/esm/locale/cs.js
+++ b/esm/locale/cs.js
@@ -1,5 +1,5 @@
 // Czech [cs]
-import dayjs from '../index';
+import dayjs from '../index.js';
 
 function plural(n) {
   return n > 1 && n < 5 && ~~(n / 10) !== 1; // eslint-disable-line
diff --git a/esm/locale/cv.js b/esm/locale/cv.js
index 7dc41f767dd03ce32726ebe1b6d51469f76621a4..670ca7ce8e7e3b34e7743e73b97225ddfb8674b7 100644
--- a/esm/locale/cv.js
+++ b/esm/locale/cv.js
@@ -1,5 +1,5 @@
 // Chuvash [cv]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'cv',
   weekdays: 'вырсарникун_тунтикун_ытларикун_юнкун_кӗҫнерникун_эрнекун_шӑматкун'.split('_'),
diff --git a/esm/locale/cy.js b/esm/locale/cy.js
index 63e6c33c2c5d51e452d1ed02e95b8e234193e6bd..249476a7e539a36b4509bfe98f2d7b9e2574f50f 100644
--- a/esm/locale/cy.js
+++ b/esm/locale/cy.js
@@ -1,5 +1,5 @@
 // Welsh [cy]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'cy',
   weekdays: 'Dydd Sul_Dydd Llun_Dydd Mawrth_Dydd Mercher_Dydd Iau_Dydd Gwener_Dydd Sadwrn'.split('_'),
diff --git a/esm/locale/da.js b/esm/locale/da.js
index 9c2d04895fbe3b18ebf78f436e922544ae3ddb9e..3e7be71787cef84a2f85d036e5681ced96e782b3 100644
--- a/esm/locale/da.js
+++ b/esm/locale/da.js
@@ -1,5 +1,5 @@
 // Danish [da]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'da',
   weekdays: 'søndag_mandag_tirsdag_onsdag_torsdag_fredag_lørdag'.split('_'),
diff --git a/esm/locale/de-at.js b/esm/locale/de-at.js
index e109d97103a6780d9e32b06f0fc99ee901a6333d..9c5b9248db3d2b2c32dc87698e8de38c02c8c11a 100644
--- a/esm/locale/de-at.js
+++ b/esm/locale/de-at.js
@@ -1,5 +1,5 @@
 // German (Austria) [de-at]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var texts = {
   s: 'ein paar Sekunden',
   m: ['eine Minute', 'einer Minute'],
diff --git a/esm/locale/de-ch.js b/esm/locale/de-ch.js
index 1ffbbf73086fa6f62ec926fa959e32b535eeecc6..45d4136ee6169dca6b1395742de433b94f155f46 100644
--- a/esm/locale/de-ch.js
+++ b/esm/locale/de-ch.js
@@ -1,5 +1,5 @@
 // German (Switzerland) [de-ch]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var texts = {
   s: 'ein paar Sekunden',
   m: ['eine Minute', 'einer Minute'],
diff --git a/esm/locale/de.js b/esm/locale/de.js
index 8ccd48337ee87cc73f37a93bc790f5ba054525ff..a92890a3664f79e3207df37299b5293ddc46b3b5 100644
--- a/esm/locale/de.js
+++ b/esm/locale/de.js
@@ -1,5 +1,5 @@
 // German [de]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var texts = {
   s: 'ein paar Sekunden',
   m: ['eine Minute', 'einer Minute'],
diff --git a/esm/locale/dv.js b/esm/locale/dv.js
index 8943fdd80c72ffac0287c3606162956b6d624f47..eb8b5a950ae7413b93237b061b0ed364a09bdaad 100644
--- a/esm/locale/dv.js
+++ b/esm/locale/dv.js
@@ -1,5 +1,5 @@
 // Maldivian [dv]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'dv',
   weekdays: 'އާދިއްތަ_ހޯމަ_އަންގާރަ_ބުދަ_ބުރާސްފަތި_ހުކުރު_ހޮނިހިރު'.split('_'),
diff --git a/esm/locale/el.js b/esm/locale/el.js
index 2aa9917dc369539b86b8209a31cf72d7f4f2910b..a79f0046daa798fdee382d1e3711b6539c081ac8 100644
--- a/esm/locale/el.js
+++ b/esm/locale/el.js
@@ -1,5 +1,5 @@
 // Greek [el]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'el',
   weekdays: 'Κυριακή_Δευτέρα_Τρίτη_Τετάρτη_Πέμπτη_Παρασκευή_Σάββατο'.split('_'),
diff --git a/esm/locale/en-au.js b/esm/locale/en-au.js
index f9dde03402bcb29d122686a468d0523c9733c241..d00770287bb7b9dc0df2cc2c862d20039841d5cc 100644
--- a/esm/locale/en-au.js
+++ b/esm/locale/en-au.js
@@ -1,5 +1,5 @@
 // English (Australia) [en-au]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-au',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/en-ca.js b/esm/locale/en-ca.js
index 8e416c9965a7eba118e692a3f05157b86164af0d..872a35e54f55c9d022afe61339fe44660b91f9d5 100644
--- a/esm/locale/en-ca.js
+++ b/esm/locale/en-ca.js
@@ -1,5 +1,5 @@
 // English (Canada) [en-ca]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-ca',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/en-gb.js b/esm/locale/en-gb.js
index f979b442fb3daf18d003d58e34cffe46261814a4..706f7d3443b62175d6c107c5c70af2521f4f8fbf 100644
--- a/esm/locale/en-gb.js
+++ b/esm/locale/en-gb.js
@@ -1,5 +1,5 @@
 // English (United Kingdom) [en-gb]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-gb',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/en-ie.js b/esm/locale/en-ie.js
index 8098d2f7a2b2c2a6779fe2937fec687d1dc70940..063213a0fdcbc9fcaa9af00eddb8b172b7d9f4cd 100644
--- a/esm/locale/en-ie.js
+++ b/esm/locale/en-ie.js
@@ -1,5 +1,5 @@
 // English (Ireland) [en-ie]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-ie',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/en-il.js b/esm/locale/en-il.js
index 56c241ac427e249a6e88320f83b032d3e7278971..5622bfeb00e63fbb75a195d07af5ef3424d5b305 100644
--- a/esm/locale/en-il.js
+++ b/esm/locale/en-il.js
@@ -1,5 +1,5 @@
 // English (Israel) [en-il]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-il',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/en-in.js b/esm/locale/en-in.js
index 7ccb206ca0e35438b83c0e9c35b352301879dbe6..43a07c2307ac8bda31bd93126da9c66b10ed000e 100644
--- a/esm/locale/en-in.js
+++ b/esm/locale/en-in.js
@@ -1,5 +1,5 @@
 // English (India) [en-in]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-in',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/en-nz.js b/esm/locale/en-nz.js
index 08c562eb02363c779c0558ec723438171b038446..f67d31ebbe6d5450f157527dc4528fdca642b66b 100644
--- a/esm/locale/en-nz.js
+++ b/esm/locale/en-nz.js
@@ -1,5 +1,5 @@
 // English (New Zealand) [en-nz]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-nz',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/en-sg.js b/esm/locale/en-sg.js
index 3c5edceef9d1a3fbf49475e0b35e7477e12b8a9c..7846202e0b810214f92104575f54baabf3863317 100644
--- a/esm/locale/en-sg.js
+++ b/esm/locale/en-sg.js
@@ -1,5 +1,5 @@
 // English (Singapore) [en-sg]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-sg',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/en-tt.js b/esm/locale/en-tt.js
index ef47eeba3af9f09b51130451f196a6c4b1334543..24d3806ee456d003be019b06a02a56300f7930f7 100644
--- a/esm/locale/en-tt.js
+++ b/esm/locale/en-tt.js
@@ -1,5 +1,5 @@
 // English (Trinidad & Tobago) [en-tt]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'en-tt',
   weekdays: 'Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday'.split('_'),
diff --git a/esm/locale/eo.js b/esm/locale/eo.js
index e62599aae764959ee5a26127dde1f62d0c9594d1..7455e04affc45afd66fb1d9c5febc01a8c6e4ddd 100644
--- a/esm/locale/eo.js
+++ b/esm/locale/eo.js
@@ -1,5 +1,5 @@
 // Esperanto [eo]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'eo',
   weekdays: 'dimanĉo_lundo_mardo_merkredo_ĵaŭdo_vendredo_sabato'.split('_'),
diff --git a/esm/locale/es-do.js b/esm/locale/es-do.js
index 09410cfab1948bd8d14aae47ed57cee21d12128e..34c1595a71379e37d232bb387298d832e6e44a63 100644
--- a/esm/locale/es-do.js
+++ b/esm/locale/es-do.js
@@ -1,5 +1,5 @@
 // Spanish (Dominican Republic) [es-do]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'es-do',
   weekdays: 'domingo_lunes_martes_miércoles_jueves_viernes_sábado'.split('_'),
diff --git a/esm/locale/es-mx.js b/esm/locale/es-mx.js
index 0207f83bac58f8951097b909d57f1cf0ce8b99ce..e39e9a7cbabfbab28c3345b0e53950f736887cff 100644
--- a/esm/locale/es-mx.js
+++ b/esm/locale/es-mx.js
@@ -1,5 +1,5 @@
 // Spanish (Mexico) [es-mx]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'es-mx',
   weekdays: 'domingo_lunes_martes_miércoles_jueves_viernes_sábado'.split('_'),
diff --git a/esm/locale/es-pr.js b/esm/locale/es-pr.js
index 5edc359eeb47697e7e197870d302482d2ed81ac1..ff209a6ce7aee5f3182e958663d54d0b98df289a 100644
--- a/esm/locale/es-pr.js
+++ b/esm/locale/es-pr.js
@@ -1,5 +1,5 @@
 // Spanish (Puerto Rico) [es-PR]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'es-pr',
   monthsShort: 'ene_feb_mar_abr_may_jun_jul_ago_sep_oct_nov_dic'.split('_'),
diff --git a/esm/locale/es-us.js b/esm/locale/es-us.js
index f9b01a08f95aea8ce113066b476e87032b2912b6..6995e2095496ac51bdc875ab93551f8c2bb9c6d9 100644
--- a/esm/locale/es-us.js
+++ b/esm/locale/es-us.js
@@ -1,5 +1,5 @@
 // Spanish (United States) [es-us]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'es-us',
   weekdays: 'domingo_lunes_martes_miércoles_jueves_viernes_sábado'.split('_'),
diff --git a/esm/locale/es.js b/esm/locale/es.js
index 84bdfbe916ce2770f35b715209a1e6b5b552baf9..050662de1f2a1fe83b7e9b399f97e3d9391f485f 100644
--- a/esm/locale/es.js
+++ b/esm/locale/es.js
@@ -1,5 +1,5 @@
 // Spanish [es]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'es',
   monthsShort: 'ene_feb_mar_abr_may_jun_jul_ago_sep_oct_nov_dic'.split('_'),
diff --git a/esm/locale/et.js b/esm/locale/et.js
index 7f7c5ff59a7d93bcd819013f957d093f37dea0fe..843da81537cefba915b29b3a38e516449635e0d5 100644
--- a/esm/locale/et.js
+++ b/esm/locale/et.js
@@ -1,5 +1,5 @@
 // Estonian [et]
-import dayjs from '../index';
+import dayjs from '../index.js';
 
 function relativeTimeWithTense(number, withoutSuffix, key, isFuture) {
   var format = {
diff --git a/esm/locale/eu.js b/esm/locale/eu.js
index 5cb73d093c38dd9dc02705ee72ad02df55a9c85a..3fdd0f6206cd1ac758c217f570daf05100182bef 100644
--- a/esm/locale/eu.js
+++ b/esm/locale/eu.js
@@ -1,5 +1,5 @@
 // Basque [eu]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'eu',
   weekdays: 'igandea_astelehena_asteartea_asteazkena_osteguna_ostirala_larunbata'.split('_'),
diff --git a/esm/locale/fa.js b/esm/locale/fa.js
index 089459e78a545933283c6b81721a2a64a12554a5..8a6f0775b073184fdbde00935f39ba100a7ec3f1 100644
--- a/esm/locale/fa.js
+++ b/esm/locale/fa.js
@@ -1,5 +1,5 @@
 // Persian [fa]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'fa',
   weekdays: 'یک‌شنبه_دوشنبه_سه‌شنبه_چهارشنبه_پنج‌شنبه_جمعه_شنبه'.split('_'),
diff --git a/esm/locale/fi.js b/esm/locale/fi.js
index 1ded894dbbb656fd4020513971c34027ecd24f29..d9139d98915dd621033ebfa957538218a807f771 100644
--- a/esm/locale/fi.js
+++ b/esm/locale/fi.js
@@ -1,5 +1,5 @@
 // Finnish [fi]
-import dayjs from '../index';
+import dayjs from '../index.js';
 
 function relativeTimeFormatter(number, withoutSuffix, key, isFuture) {
   var past = {
diff --git a/esm/locale/fo.js b/esm/locale/fo.js
index 07c376173d8d2cec9a93ba1d13013bad36865414..cdfb5efda6b7fdd6f65724fa87889de7e3337f1e 100644
--- a/esm/locale/fo.js
+++ b/esm/locale/fo.js
@@ -1,5 +1,5 @@
 // Faroese [fo]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'fo',
   weekdays: 'sunnudagur_mánadagur_týsdagur_mikudagur_hósdagur_fríggjadagur_leygardagur'.split('_'),
diff --git a/esm/locale/fr-ca.js b/esm/locale/fr-ca.js
index 688d695c67b0ba678c1441dafbd31bcb54af5c68..eea2a1425414fecc87355899c7a1f5ad49d73e4b 100644
--- a/esm/locale/fr-ca.js
+++ b/esm/locale/fr-ca.js
@@ -1,5 +1,5 @@
 // French (Canada) [fr-ca]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'fr-ca',
   weekdays: 'dimanche_lundi_mardi_mercredi_jeudi_vendredi_samedi'.split('_'),
diff --git a/esm/locale/fr-ch.js b/esm/locale/fr-ch.js
index 593dba804685b9220c6a4d91b856fe5a38697b1d..76796e280c9c2a89c16f178ac79f7a39639b5229 100644
--- a/esm/locale/fr-ch.js
+++ b/esm/locale/fr-ch.js
@@ -1,5 +1,5 @@
 // French (Switzerland) [fr-ch]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'fr-ch',
   weekdays: 'dimanche_lundi_mardi_mercredi_jeudi_vendredi_samedi'.split('_'),
diff --git a/esm/locale/fr.js b/esm/locale/fr.js
index b31c11d3ad848684c246edfdca9529566b9ad0d3..eee9ae261720cde188fa50278bcb1f6a3a729b51 100644
--- a/esm/locale/fr.js
+++ b/esm/locale/fr.js
@@ -1,5 +1,5 @@
 // French [fr]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'fr',
   weekdays: 'dimanche_lundi_mardi_mercredi_jeudi_vendredi_samedi'.split('_'),
diff --git a/esm/locale/fy.js b/esm/locale/fy.js
index 4b9f9de41ec829449009f3f68fc9e5f524084b8b..30c26c5217b7885e28f0f485a579d1c2a44a91f8 100644
--- a/esm/locale/fy.js
+++ b/esm/locale/fy.js
@@ -1,5 +1,5 @@
 // Frisian [fy]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'fy',
   weekdays: 'snein_moandei_tiisdei_woansdei_tongersdei_freed_sneon'.split('_'),
diff --git a/esm/locale/ga.js b/esm/locale/ga.js
index 8cdfa9f128db4631c65a6f6ad2ac0016f4191d10..6f5deb3594458dc6187040ad0c27e1ccdefc9abb 100644
--- a/esm/locale/ga.js
+++ b/esm/locale/ga.js
@@ -1,5 +1,5 @@
 // Irish or Irish Gaelic [ga]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ga',
   weekdays: 'Dé Domhnaigh_Dé Luain_Dé Máirt_Dé Céadaoin_Déardaoin_Dé hAoine_Dé Satharn'.split('_'),
diff --git a/esm/locale/gd.js b/esm/locale/gd.js
index fcf62cd5d769f76b0ba73a9b097f82d79487d639..fbe760bdebf12fa7b8c872bd1c0bc942edb0043e 100644
--- a/esm/locale/gd.js
+++ b/esm/locale/gd.js
@@ -1,5 +1,5 @@
 // Scottish Gaelic [gd]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'gd',
   weekdays: 'Didòmhnaich_Diluain_Dimàirt_Diciadain_Diardaoin_Dihaoine_Disathairne'.split('_'),
diff --git a/esm/locale/gl.js b/esm/locale/gl.js
index 23d687f634adfcee60c2db32101fdf7ab1f0a824..95831306d5ab770337134f37e7718c2898329185 100644
--- a/esm/locale/gl.js
+++ b/esm/locale/gl.js
@@ -1,5 +1,5 @@
 // Galician [gl]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'gl',
   weekdays: 'domingo_luns_martes_mércores_xoves_venres_sábado'.split('_'),
diff --git a/esm/locale/gom-latn.js b/esm/locale/gom-latn.js
index d621f5b7614311b08bc1d62f61cf80a5508486cd..028a6c9c717dc876f235d569ee19357477553d1c 100644
--- a/esm/locale/gom-latn.js
+++ b/esm/locale/gom-latn.js
@@ -1,5 +1,5 @@
 // Konkani Latin script [gom-latn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'gom-latn',
   weekdays: "Aitar_Somar_Mongllar_Budvar_Brestar_Sukrar_Son'var".split('_'),
diff --git a/esm/locale/gu.js b/esm/locale/gu.js
index e05f44ba1a7cf3a3c94ede2305b6f654eee16201..2a65793f9d6e73194a109921f35256a3fafc7a72 100644
--- a/esm/locale/gu.js
+++ b/esm/locale/gu.js
@@ -1,5 +1,5 @@
 // Gujarati [gu]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'gu',
   weekdays: 'રવિવાર_સોમવાર_મંગળવાર_બુધ્વાર_ગુરુવાર_શુક્રવાર_શનિવાર'.split('_'),
diff --git a/esm/locale/he.js b/esm/locale/he.js
index a8868ff69802c411c8b2ceae2af09081502b478c..f65b5cd48b56fbdc79abeb32cb583cea8da3a438 100644
--- a/esm/locale/he.js
+++ b/esm/locale/he.js
@@ -1,5 +1,5 @@
 // Hebrew [he]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var texts = {
   s: 'מספר שניות',
   ss: '%d שניות',
diff --git a/esm/locale/hi.js b/esm/locale/hi.js
index e877ed6b1033d95ad73f8b030a147b2110bb5ea5..425bfb8b008707850354ff26609e3d45348a5198 100644
--- a/esm/locale/hi.js
+++ b/esm/locale/hi.js
@@ -1,5 +1,5 @@
 // Hindi [hi]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'hi',
   weekdays: 'रविवार_सोमवार_मंगलवार_बुधवार_गुरूवार_शुक्रवार_शनिवार'.split('_'),
diff --git a/esm/locale/hr.js b/esm/locale/hr.js
index a760fe34789ccae60f22a7133181438226568255..ba0854e9af7295bebb9a338babf7d40317a7f724 100644
--- a/esm/locale/hr.js
+++ b/esm/locale/hr.js
@@ -1,5 +1,5 @@
 // Croatian [hr]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var monthFormat = 'siječnja_veljače_ožujka_travnja_svibnja_lipnja_srpnja_kolovoza_rujna_listopada_studenoga_prosinca'.split('_');
 var monthStandalone = 'siječanj_veljača_ožujak_travanj_svibanj_lipanj_srpanj_kolovoz_rujan_listopad_studeni_prosinac'.split('_');
 var MONTHS_IN_FORMAT = /D[oD]?(\[[^[\]]*\]|\s)+MMMM?/;
diff --git a/esm/locale/ht.js b/esm/locale/ht.js
index 896739e36ce148de5a285e30ad27bb5995658730..cd8ac74ac516444bcb0c7d56f59cd2b3a294d529 100644
--- a/esm/locale/ht.js
+++ b/esm/locale/ht.js
@@ -1,5 +1,5 @@
 // Haitian Creole (Haiti) [ht]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ht',
   weekdays: 'dimanch_lendi_madi_mèkredi_jedi_vandredi_samdi'.split('_'),
diff --git a/esm/locale/hu.js b/esm/locale/hu.js
index 18df6e46a89afa2f6ffd226bf2194435c7f92099..7825da055283221df7e0932d6877d2422073c514 100644
--- a/esm/locale/hu.js
+++ b/esm/locale/hu.js
@@ -1,5 +1,5 @@
 // Hungarian [hu]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'hu',
   weekdays: 'vasárnap_hétfő_kedd_szerda_csütörtök_péntek_szombat'.split('_'),
diff --git a/esm/locale/hy-am.js b/esm/locale/hy-am.js
index 937f2be5e9d7f967e88e9874abbf6bc7b7beed71..6e633fb6e53acd07788d7fe000337a62cc5d13d5 100644
--- a/esm/locale/hy-am.js
+++ b/esm/locale/hy-am.js
@@ -1,5 +1,5 @@
 // Armenian [hy-am]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'hy-am',
   weekdays: 'կիրակի_երկուշաբթի_երեքշաբթի_չորեքշաբթի_հինգշաբթի_ուրբաթ_շաբաթ'.split('_'),
diff --git a/esm/locale/id.js b/esm/locale/id.js
index f743a123cfdb50705284ac453bec4a9867660e9c..d875e4e613ea96532064065e5950c34851e7e48b 100644
--- a/esm/locale/id.js
+++ b/esm/locale/id.js
@@ -1,5 +1,5 @@
 // Indonesian [id]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'id',
   weekdays: 'Minggu_Senin_Selasa_Rabu_Kamis_Jumat_Sabtu'.split('_'),
diff --git a/esm/locale/is.js b/esm/locale/is.js
index 22d81212910be9fc169f68c196bca0899400d380..9a1a60e73c53aa2a5d6e83c982fd7152486438bf 100644
--- a/esm/locale/is.js
+++ b/esm/locale/is.js
@@ -1,5 +1,5 @@
 // Icelandic [is]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var texts = {
   s: ['nokkrar sekúndur', 'nokkrar sekúndur', 'nokkrum sekúndum'],
   m: ['mínúta', 'mínútu', 'mínútu'],
diff --git a/esm/locale/it-ch.js b/esm/locale/it-ch.js
index cfbb94dd4fa8601cc03a7f516adcbdee9ec49244..bb8e4ce6f050ccdf007ab9aa46b1f7c87a3f175e 100644
--- a/esm/locale/it-ch.js
+++ b/esm/locale/it-ch.js
@@ -1,5 +1,5 @@
 // Italian (Switzerland) [it-ch]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'it-ch',
   weekdays: 'domenica_lunedì_martedì_mercoledì_giovedì_venerdì_sabato'.split('_'),
diff --git a/esm/locale/it.js b/esm/locale/it.js
index e8d24905894d244ddd72c93cd9011d7302b01c82..ae892010adf96dcb0d96aa8e29143a1d7001fcf8 100644
--- a/esm/locale/it.js
+++ b/esm/locale/it.js
@@ -1,5 +1,5 @@
 // Italian [it]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'it',
   weekdays: 'domenica_lunedì_martedì_mercoledì_giovedì_venerdì_sabato'.split('_'),
diff --git a/esm/locale/ja.js b/esm/locale/ja.js
index 6568e1390856f43115382f5937437dcb23dd7bb7..0a31f9278b18036f3875e054d997b7f28f7d6205 100644
--- a/esm/locale/ja.js
+++ b/esm/locale/ja.js
@@ -1,5 +1,5 @@
 // Japanese [ja]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ja',
   weekdays: '日曜日_月曜日_火曜日_水曜日_木曜日_金曜日_土曜日'.split('_'),
diff --git a/esm/locale/jv.js b/esm/locale/jv.js
index 81a3f6607475628b91b64b2e46cbd7f93b700332..6f56885faa6400725edf102ebcfde516c500cb31 100644
--- a/esm/locale/jv.js
+++ b/esm/locale/jv.js
@@ -1,5 +1,5 @@
 // Javanese [jv]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'jv',
   weekdays: 'Minggu_Senen_Seloso_Rebu_Kemis_Jemuwah_Septu'.split('_'),
diff --git a/esm/locale/ka.js b/esm/locale/ka.js
index 381fffa4168c8ad883076df8add2de5ef993f1b6..1c149e4d508f4a326b4a7772191005261931ba46 100644
--- a/esm/locale/ka.js
+++ b/esm/locale/ka.js
@@ -1,5 +1,5 @@
 // Georgian [ka]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ka',
   weekdays: 'კვირა_ორშაბათი_სამშაბათი_ოთხშაბათი_ხუთშაბათი_პარასკევი_შაბათი'.split('_'),
diff --git a/esm/locale/kk.js b/esm/locale/kk.js
index f2ca045c3154185989ade5e4b47a1c9c53f25f81..fcf300fb84b038f798bbfcef37a2efb822d16ed7 100644
--- a/esm/locale/kk.js
+++ b/esm/locale/kk.js
@@ -1,5 +1,5 @@
 // Kazakh [kk]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'kk',
   weekdays: 'жексенбі_дүйсенбі_сейсенбі_сәрсенбі_бейсенбі_жұма_сенбі'.split('_'),
diff --git a/esm/locale/km.js b/esm/locale/km.js
index 7fd185ba2f6805abbf055af67031e1bbdfd7b5de..ea44972576e39346e546181a3b709a3714e13455 100644
--- a/esm/locale/km.js
+++ b/esm/locale/km.js
@@ -1,5 +1,5 @@
 // Cambodian [km]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'km',
   weekdays: 'អាទិត្យ_ច័ន្ទ_អង្គារ_ពុធ_ព្រហស្បតិ៍_សុក្រ_សៅរ៍'.split('_'),
diff --git a/esm/locale/kn.js b/esm/locale/kn.js
index b9ca9b9084c22c14002930415bfe564865dc0bb3..94b4841fe7f47077c6866e60a307b92bdf248b88 100644
--- a/esm/locale/kn.js
+++ b/esm/locale/kn.js
@@ -1,5 +1,5 @@
 // Kannada [kn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'kn',
   weekdays: 'ಭಾನುವಾರ_ಸೋಮವಾರ_ಮಂಗಳವಾರ_ಬುಧವಾರ_ಗುರುವಾರ_ಶುಕ್ರವಾರ_ಶನಿವಾರ'.split('_'),
diff --git a/esm/locale/ko.js b/esm/locale/ko.js
index cfad49de21da56f7c2069a73be2c5a6b07dfcb2f..a501a74e215be0036b0d613c477a364b2d6570a3 100644
--- a/esm/locale/ko.js
+++ b/esm/locale/ko.js
@@ -1,5 +1,5 @@
 // Korean [ko]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ko',
   weekdays: '일요일_월요일_화요일_수요일_목요일_금요일_토요일'.split('_'),
diff --git a/esm/locale/ku.js b/esm/locale/ku.js
index e56664eb8f7432f3aa15ec97939de55ddc2f9fc1..56283aaf79671c527d1d5bf5bcd983b33efbcb66 100644
--- a/esm/locale/ku.js
+++ b/esm/locale/ku.js
@@ -1,5 +1,5 @@
 // Kurdish [ku]
-import dayjs from '../index';
+import dayjs from '../index.js';
 export var englishToArabicNumbersMap = {
   1: '١',
   2: '٢',
diff --git a/esm/locale/ky.js b/esm/locale/ky.js
index fd044778b312a5c48a77c593cd3aaf6d002a1e13..eae7e2703680612fb1f659189efedc97cc3931ea 100644
--- a/esm/locale/ky.js
+++ b/esm/locale/ky.js
@@ -1,5 +1,5 @@
 // Kyrgyz [ky]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ky',
   weekdays: 'Жекшемби_Дүйшөмбү_Шейшемби_Шаршемби_Бейшемби_Жума_Ишемби'.split('_'),
diff --git a/esm/locale/lb.js b/esm/locale/lb.js
index 21ef4aab68ad992a33f051036b6a120019b3ea9e..3f14f70ca8a0b9950d4173fd573d3cb676aceb2d 100644
--- a/esm/locale/lb.js
+++ b/esm/locale/lb.js
@@ -1,5 +1,5 @@
 // Luxembourgish [lb]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'lb',
   weekdays: 'Sonndeg_Méindeg_Dënschdeg_Mëttwoch_Donneschdeg_Freideg_Samschdeg'.split('_'),
diff --git a/esm/locale/lo.js b/esm/locale/lo.js
index 7732ec4b17f86e79d9ad98a5cd0740561da3525c..58aa37a9842cacc56a5a67232c2373f15fd47536 100644
--- a/esm/locale/lo.js
+++ b/esm/locale/lo.js
@@ -1,5 +1,5 @@
 // Lao [lo]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'lo',
   weekdays: 'ອາທິດ_ຈັນ_ອັງຄານ_ພຸດ_ພະຫັດ_ສຸກ_ເສົາ'.split('_'),
diff --git a/esm/locale/lt.js b/esm/locale/lt.js
index cb46ca9b8ce8e2088d89f58e9c03c2cd14f3ae12..bc5bb74ee6875c4573fd6519dee9ba62b7e91ebf 100644
--- a/esm/locale/lt.js
+++ b/esm/locale/lt.js
@@ -1,5 +1,5 @@
 // Lithuanian [lt]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var monthFormat = 'sausio_vasario_kovo_balandžio_gegužės_birželio_liepos_rugpjūčio_rugsėjo_spalio_lapkričio_gruodžio'.split('_');
 var monthStandalone = 'sausis_vasaris_kovas_balandis_gegužė_birželis_liepa_rugpjūtis_rugsėjis_spalis_lapkritis_gruodis'.split('_'); // eslint-disable-next-line no-useless-escape
 
diff --git a/esm/locale/lv.js b/esm/locale/lv.js
index 4b18a61403b8c41442a5859369acdea8474a1eda..89c4274b3f87978fedc6c9c6d3423c33d2215b78 100644
--- a/esm/locale/lv.js
+++ b/esm/locale/lv.js
@@ -1,5 +1,5 @@
 // Latvian [lv]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'lv',
   weekdays: 'svētdiena_pirmdiena_otrdiena_trešdiena_ceturtdiena_piektdiena_sestdiena'.split('_'),
diff --git a/esm/locale/me.js b/esm/locale/me.js
index 465c0ff0478ac441749d7398e99ad95b91d594c3..40369041ea466a92461d41e27eea000627a6b29b 100644
--- a/esm/locale/me.js
+++ b/esm/locale/me.js
@@ -1,5 +1,5 @@
 // Montenegrin [me]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'me',
   weekdays: 'nedjelja_ponedjeljak_utorak_srijeda_četvrtak_petak_subota'.split('_'),
diff --git a/esm/locale/mi.js b/esm/locale/mi.js
index 3b56f0ec7fd44d507f2e0154bd63973bc17f7b89..fc630f99b70212236ded0875339821aa8019ec3e 100644
--- a/esm/locale/mi.js
+++ b/esm/locale/mi.js
@@ -1,5 +1,5 @@
 // Maori [mi]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'mi',
   weekdays: 'Rātapu_Mane_Tūrei_Wenerei_Tāite_Paraire_Hātarei'.split('_'),
diff --git a/esm/locale/mk.js b/esm/locale/mk.js
index 8522c261ca1910562a209c882406bbd243eaf581..1c000c45c6a0c22ce9cb96bfb4619bd4be018adb 100644
--- a/esm/locale/mk.js
+++ b/esm/locale/mk.js
@@ -1,5 +1,5 @@
 // Macedonian [mk]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'mk',
   weekdays: 'недела_понеделник_вторник_среда_четврток_петок_сабота'.split('_'),
diff --git a/esm/locale/ml.js b/esm/locale/ml.js
index bfcc277eb076dd258dc79b3e2897bcb3a758ca0b..bd5af14564beea8d5779c15a30e7e4998b110392 100644
--- a/esm/locale/ml.js
+++ b/esm/locale/ml.js
@@ -1,5 +1,5 @@
 // Malayalam [ml]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ml',
   weekdays: 'ഞായറാഴ്ച_തിങ്കളാഴ്ച_ചൊവ്വാഴ്ച_ബുധനാഴ്ച_വ്യാഴാഴ്ച_വെള്ളിയാഴ്ച_ശനിയാഴ്ച'.split('_'),
diff --git a/esm/locale/mn.js b/esm/locale/mn.js
index d93cae216da8814393685342cde3e844e8c016ac..654c1cf5d1085531f5ec532f3d8fab775e0828fe 100644
--- a/esm/locale/mn.js
+++ b/esm/locale/mn.js
@@ -1,5 +1,5 @@
 // Mongolian [mn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'mn',
   weekdays: 'Ням_Даваа_Мягмар_Лхагва_Пүрэв_Баасан_Бямба'.split('_'),
diff --git a/esm/locale/mr.js b/esm/locale/mr.js
index 9eac8a734a6d7d4d106f633b0962205865a77680..30d615137725ae580a6f69c58e76664c442197a8 100644
--- a/esm/locale/mr.js
+++ b/esm/locale/mr.js
@@ -1,5 +1,5 @@
 // Marathi [mr]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'mr',
   weekdays: 'रविवार_सोमवार_मंगळवार_बुधवार_गुरूवार_शुक्रवार_शनिवार'.split('_'),
diff --git a/esm/locale/ms-my.js b/esm/locale/ms-my.js
index 51382197a892a1d22c1e763bd10429beb32a84e3..a6bae2dcc5fb7390c8f4dd7705a78d65d4797506 100644
--- a/esm/locale/ms-my.js
+++ b/esm/locale/ms-my.js
@@ -1,5 +1,5 @@
 // Malay [ms-my]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ms-my',
   weekdays: 'Ahad_Isnin_Selasa_Rabu_Khamis_Jumaat_Sabtu'.split('_'),
diff --git a/esm/locale/ms.js b/esm/locale/ms.js
index 86349f3fde70b8d7f7a0561c69b3992da4909dd9..c0849f9d8e1fe6f934adfa72fc21bca9fa3e1fff 100644
--- a/esm/locale/ms.js
+++ b/esm/locale/ms.js
@@ -1,5 +1,5 @@
 // Malay [ms]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ms',
   weekdays: 'Ahad_Isnin_Selasa_Rabu_Khamis_Jumaat_Sabtu'.split('_'),
diff --git a/esm/locale/mt.js b/esm/locale/mt.js
index 9c909538121bda4baf95c1da60f48e55371d4da8..6e4de19ca3b8443d624b19811c57d78abea03ef7 100644
--- a/esm/locale/mt.js
+++ b/esm/locale/mt.js
@@ -1,5 +1,5 @@
 // Maltese (Malta) [mt]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'mt',
   weekdays: 'Il-Ħadd_It-Tnejn_It-Tlieta_L-Erbgħa_Il-Ħamis_Il-Ġimgħa_Is-Sibt'.split('_'),
diff --git a/esm/locale/my.js b/esm/locale/my.js
index 73b263339ea06c5fbf04c09f3d5447573aa7c1fe..66aa40d31af4ca734108dc8a11e23205ca7ce5c9 100644
--- a/esm/locale/my.js
+++ b/esm/locale/my.js
@@ -1,5 +1,5 @@
 // Burmese [my]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'my',
   weekdays: 'တနင်္ဂနွေ_တနင်္လာ_အင်္ဂါ_ဗုဒ္ဓဟူး_ကြာသပတေး_သောကြာ_စနေ'.split('_'),
diff --git a/esm/locale/nb.js b/esm/locale/nb.js
index 1d7b1eb56e660f5ef1b09f2b9a214025eeb4f70b..6e142fe1b54ee11d2c7bfc31140e681018485529 100644
--- a/esm/locale/nb.js
+++ b/esm/locale/nb.js
@@ -1,5 +1,5 @@
 // Norwegian Bokmål [nb]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'nb',
   weekdays: 'søndag_mandag_tirsdag_onsdag_torsdag_fredag_lørdag'.split('_'),
diff --git a/esm/locale/ne.js b/esm/locale/ne.js
index 4f5a0044e89607827fb3a0bc9da9ad1efb82bcdf..05242e49b8f71e05a958c6508e6836ddd6aa0c97 100644
--- a/esm/locale/ne.js
+++ b/esm/locale/ne.js
@@ -1,5 +1,5 @@
 // Nepalese [ne]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ne',
   weekdays: 'आइतबार_सोमबार_मङ्गलबार_बुधबार_बिहिबार_शुक्रबार_शनिबार'.split('_'),
diff --git a/esm/locale/nl-be.js b/esm/locale/nl-be.js
index 51465b7101c2a621e4ccc95195bb46f341795ed9..754fbda85f63ba47d6068fbbb931c959f25d41af 100644
--- a/esm/locale/nl-be.js
+++ b/esm/locale/nl-be.js
@@ -1,5 +1,5 @@
 // Dutch (Belgium) [nl-be]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'nl-be',
   weekdays: 'zondag_maandag_dinsdag_woensdag_donderdag_vrijdag_zaterdag'.split('_'),
diff --git a/esm/locale/nl.js b/esm/locale/nl.js
index ee1ac747a46ecbbff266c4c119d44042859596a0..f4492a4775271de156b12e77101ce46c338fd4d4 100644
--- a/esm/locale/nl.js
+++ b/esm/locale/nl.js
@@ -1,5 +1,5 @@
 // Dutch [nl]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'nl',
   weekdays: 'zondag_maandag_dinsdag_woensdag_donderdag_vrijdag_zaterdag'.split('_'),
diff --git a/esm/locale/nn.js b/esm/locale/nn.js
index 43767a40bb312e64710977d99aafe1ce6b872711..1bbaf9cdd955618d8dfab1a3d501f72e684d7d33 100644
--- a/esm/locale/nn.js
+++ b/esm/locale/nn.js
@@ -1,5 +1,5 @@
 // Nynorsk [nn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'nn',
   weekdays: 'sundag_måndag_tysdag_onsdag_torsdag_fredag_laurdag'.split('_'),
diff --git a/esm/locale/oc-lnc.js b/esm/locale/oc-lnc.js
index 91e2f0d35252198f710a800cdf1128dad1360eeb..3f64973fb8f6525471eb37190a649e0f3dfce9e1 100644
--- a/esm/locale/oc-lnc.js
+++ b/esm/locale/oc-lnc.js
@@ -1,5 +1,5 @@
 // Occitan, lengadocian dialecte [oc-lnc]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'oc-lnc',
   weekdays: 'dimenge_diluns_dimars_dimècres_dijòus_divendres_dissabte'.split('_'),
diff --git a/esm/locale/pa-in.js b/esm/locale/pa-in.js
index 624a8522ee83b6b804549aa81a426d4ecb16f3eb..528fdd6148b3617edec17af02867467be2155edb 100644
--- a/esm/locale/pa-in.js
+++ b/esm/locale/pa-in.js
@@ -1,5 +1,5 @@
 // Punjabi (India) [pa-in]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'pa-in',
   weekdays: 'ਐਤਵਾਰ_ਸੋਮਵਾਰ_ਮੰਗਲਵਾਰ_ਬੁਧਵਾਰ_ਵੀਰਵਾਰ_ਸ਼ੁੱਕਰਵਾਰ_ਸ਼ਨੀਚਰਵਾਰ'.split('_'),
diff --git a/esm/locale/pl.js b/esm/locale/pl.js
index 368b2a5e83eae2eacbc2de8131c05026dcaaae1a..535374907581529383ba7439233c2b5874156b01 100644
--- a/esm/locale/pl.js
+++ b/esm/locale/pl.js
@@ -1,5 +1,5 @@
 // Polish [pl]
-import dayjs from '../index';
+import dayjs from '../index.js';
 
 function plural(n) {
   return n % 10 < 5 && n % 10 > 1 && ~~(n / 10) % 10 !== 1; // eslint-disable-line
diff --git a/esm/locale/pt-br.js b/esm/locale/pt-br.js
index 0635cd850c4719e48742ed775e5260ea23b55389..a6a488aba88e176864b9f8670418ead1b9cdb3ab 100644
--- a/esm/locale/pt-br.js
+++ b/esm/locale/pt-br.js
@@ -1,5 +1,5 @@
 // Portuguese (Brazil) [pt-br]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'pt-br',
   weekdays: 'domingo_segunda-feira_terça-feira_quarta-feira_quinta-feira_sexta-feira_sábado'.split('_'),
diff --git a/esm/locale/pt.js b/esm/locale/pt.js
index cba233197b2ee9ae33b9fcb9705e4f0d601fc793..7b397b513e877a492ef5c74c338ca2f6b88f74c1 100644
--- a/esm/locale/pt.js
+++ b/esm/locale/pt.js
@@ -1,5 +1,5 @@
 // Portuguese [pt]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'pt',
   weekdays: 'domingo_segunda-feira_terça-feira_quarta-feira_quinta-feira_sexta-feira_sábado'.split('_'),
diff --git a/esm/locale/rn.js b/esm/locale/rn.js
index 21b3cdb732f43cbdf95a1716dfe3a13caeca0ca7..341e7d8518d14c1f32eac2b0aac43a5ebbfc8202 100644
--- a/esm/locale/rn.js
+++ b/esm/locale/rn.js
@@ -1,5 +1,5 @@
 // Kirundi [rn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'rn',
   weekdays: 'Ku wa Mungu_Ku wa Mbere_Ku wa Kabiri_Ku wa Gatatu_Ku wa Kane_Ku wa Gatanu_Ku wa Gatandatu'.split('_'),
diff --git a/esm/locale/ro.js b/esm/locale/ro.js
index 93ef6bfc1449313c703142be3bb292c6cf48ef5f..626fa346e58169330fee6d84baa949ba50edf218 100644
--- a/esm/locale/ro.js
+++ b/esm/locale/ro.js
@@ -1,5 +1,5 @@
 // Romanian [ro]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ro',
   weekdays: 'Duminică_Luni_Marți_Miercuri_Joi_Vineri_Sâmbătă'.split('_'),
diff --git a/esm/locale/ru.js b/esm/locale/ru.js
index fbb1b358917b2f03885aec0342c12325da1b4311..6c155d23f185e56da0f11979163942edffb59ae3 100644
--- a/esm/locale/ru.js
+++ b/esm/locale/ru.js
@@ -1,5 +1,5 @@
 // Russian [ru]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var monthFormat = 'января_февраля_марта_апреля_мая_июня_июля_августа_сентября_октября_ноября_декабря'.split('_');
 var monthStandalone = 'январь_февраль_март_апрель_май_июнь_июль_август_сентябрь_октябрь_ноябрь_декабрь'.split('_');
 var monthShortFormat = 'янв._февр._мар._апр._мая_июня_июля_авг._сент._окт._нояб._дек.'.split('_');
diff --git a/esm/locale/rw.js b/esm/locale/rw.js
index 1e53ac7600a1168c01e32afb5b1d7c105038c6ed..a8ff4715f5deb7c9a265dd9f0037c4c0f817740f 100644
--- a/esm/locale/rw.js
+++ b/esm/locale/rw.js
@@ -1,5 +1,5 @@
 // Kinyarwanda (Rwanda) [rw]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'rw',
   weekdays: 'Ku Cyumweru_Kuwa Mbere_Kuwa Kabiri_Kuwa Gatatu_Kuwa Kane_Kuwa Gatanu_Kuwa Gatandatu'.split('_'),
diff --git a/esm/locale/sd.js b/esm/locale/sd.js
index a429f8db298553a8e9f393273553b4fbe677a3cb..568f4e78e645a6a865d2c0f3e9b9cf26cdc07a7a 100644
--- a/esm/locale/sd.js
+++ b/esm/locale/sd.js
@@ -1,5 +1,5 @@
 // Sindhi [sd]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'sd',
   weekdays: 'آچر_سومر_اڱارو_اربع_خميس_جمع_ڇنڇر'.split('_'),
diff --git a/esm/locale/se.js b/esm/locale/se.js
index 691099cef18e3f70fe6df837890ddd676d2c32c3..2631157236ad33ef6e300317360e4663d36de738 100644
--- a/esm/locale/se.js
+++ b/esm/locale/se.js
@@ -1,5 +1,5 @@
 // Northern Sami [se]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'se',
   weekdays: 'sotnabeaivi_vuossárga_maŋŋebárga_gaskavahkku_duorastat_bearjadat_lávvardat'.split('_'),
diff --git a/esm/locale/si.js b/esm/locale/si.js
index 89b67bfd1fae60c1998d32667dc873b0fd9de506..54f908196cb5530c9d56c13b79a73f3c75cb096d 100644
--- a/esm/locale/si.js
+++ b/esm/locale/si.js
@@ -1,5 +1,5 @@
 // Sinhalese [si]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'si',
   weekdays: 'ඉරිදා_සඳුදා_අඟහරුවාදා_බදාදා_බ්‍රහස්පතින්දා_සිකුරාදා_සෙනසුරාදා'.split('_'),
diff --git a/esm/locale/sk.js b/esm/locale/sk.js
index 222401fd5894f6c3ea56765e7b7e577e9c6cad15..6f626b0d881e5f833681abc070c7471d430e53f7 100644
--- a/esm/locale/sk.js
+++ b/esm/locale/sk.js
@@ -1,5 +1,5 @@
 // Slovak [sk]
-import dayjs from '../index';
+import dayjs from '../index.js';
 
 function plural(n) {
   return n > 1 && n < 5 && ~~(n / 10) !== 1; // eslint-disable-line
diff --git a/esm/locale/sl.js b/esm/locale/sl.js
index e3c583935502eabf2ac8b1ed96916457743ff2ff..a73972d5c9630d163fd2d4d9499da1863b9c2c72 100644
--- a/esm/locale/sl.js
+++ b/esm/locale/sl.js
@@ -1,5 +1,5 @@
 // Slovenian [sl]
-import dayjs from '../index';
+import dayjs from '../index.js';
 
 function dual(n) {
   return n % 100 == 2; // eslint-disable-line
diff --git a/esm/locale/sq.js b/esm/locale/sq.js
index 625b7013d93340291bbd126ebb644447cb4b5488..3688c3cc8bd85e8ea670a8486c450a467671a6f0 100644
--- a/esm/locale/sq.js
+++ b/esm/locale/sq.js
@@ -1,5 +1,5 @@
 // Albanian [sq]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'sq',
   weekdays: 'E Diel_E Hënë_E Martë_E Mërkurë_E Enjte_E Premte_E Shtunë'.split('_'),
diff --git a/esm/locale/sr-cyrl.js b/esm/locale/sr-cyrl.js
index 2e40d51acc1aa271657d65cdb7fdd42c3399fedf..283e4b25c3481985d4dca643ff50801bc3b382b5 100644
--- a/esm/locale/sr-cyrl.js
+++ b/esm/locale/sr-cyrl.js
@@ -1,5 +1,5 @@
 // Serbian Cyrillic [sr-cyrl]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var translator = {
   words: {
     m: ['један минут', 'једног минута'],
diff --git a/esm/locale/sr.js b/esm/locale/sr.js
index f5174cedbf56de80eef7a0d3795ec815d0a495bf..4fb6e885c2ea7295c12ffa280bb19cae92eb4a71 100644
--- a/esm/locale/sr.js
+++ b/esm/locale/sr.js
@@ -1,5 +1,5 @@
 // Serbian [sr]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var translator = {
   words: {
     m: ['jedan minut', 'jednog minuta'],
diff --git a/esm/locale/ss.js b/esm/locale/ss.js
index 4354a4843d262add5fdefd5c3a9c01e45550dce9..2b612a18f25ad0a4dd8e8be14cea6cb6600a85dc 100644
--- a/esm/locale/ss.js
+++ b/esm/locale/ss.js
@@ -1,5 +1,5 @@
 // siSwati [ss]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ss',
   weekdays: 'Lisontfo_Umsombuluko_Lesibili_Lesitsatfu_Lesine_Lesihlanu_Umgcibelo'.split('_'),
diff --git a/esm/locale/sv-fi.js b/esm/locale/sv-fi.js
index a18977fa0689c06bc5b2098cf8719a7eacc661c3..9a8e16185ace1821faa44c896e77b65ab5f3b9c3 100644
--- a/esm/locale/sv-fi.js
+++ b/esm/locale/sv-fi.js
@@ -1,5 +1,5 @@
 // Finland Swedish [sv-fi]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'sv-fi',
   weekdays: 'söndag_måndag_tisdag_onsdag_torsdag_fredag_lördag'.split('_'),
diff --git a/esm/locale/sv.js b/esm/locale/sv.js
index 2563ee7c91e2c56d655f1ce28b96c5d29d302d3a..bb5107e87e7b2997387a0c81a0c27b16b9af5f8f 100644
--- a/esm/locale/sv.js
+++ b/esm/locale/sv.js
@@ -1,5 +1,5 @@
 // Swedish [sv]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'sv',
   weekdays: 'söndag_måndag_tisdag_onsdag_torsdag_fredag_lördag'.split('_'),
diff --git a/esm/locale/sw.js b/esm/locale/sw.js
index 287bf3360b3454bec896a486cd459d95c5d64d75..a17fac057ce926351b81838ea1ba03344552e3b1 100644
--- a/esm/locale/sw.js
+++ b/esm/locale/sw.js
@@ -1,5 +1,5 @@
 // Swahili [sw]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'sw',
   weekdays: 'Jumapili_Jumatatu_Jumanne_Jumatano_Alhamisi_Ijumaa_Jumamosi'.split('_'),
diff --git a/esm/locale/ta.js b/esm/locale/ta.js
index 6df25f876821b9494e54d605766e24422b8bf6f8..af8e4a3f486a489f273df2e030450cfcdc9ca53f 100644
--- a/esm/locale/ta.js
+++ b/esm/locale/ta.js
@@ -1,5 +1,5 @@
 // Tamil [ta]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ta',
   weekdays: 'ஞாயிற்றுக்கிழமை_திங்கட்கிழமை_செவ்வாய்கிழமை_புதன்கிழமை_வியாழக்கிழமை_வெள்ளிக்கிழமை_சனிக்கிழமை'.split('_'),
diff --git a/esm/locale/te.js b/esm/locale/te.js
index 392a2478c290b5b174b4beed54369cb9e4c74c4c..fc5e1a8c8b6e0c938118ea84a7d5a25705422b9b 100644
--- a/esm/locale/te.js
+++ b/esm/locale/te.js
@@ -1,5 +1,5 @@
 // Telugu [te]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'te',
   weekdays: 'ఆదివారం_సోమవారం_మంగళవారం_బుధవారం_గురువారం_శుక్రవారం_శనివారం'.split('_'),
diff --git a/esm/locale/tet.js b/esm/locale/tet.js
index ff83eeab05f267cf815ea65207625c6af042f443..31c6664e5f5868c5c149a8dc05f2bba793d9c338 100644
--- a/esm/locale/tet.js
+++ b/esm/locale/tet.js
@@ -1,5 +1,5 @@
 // Tetun Dili (East Timor) [tet]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tet',
   weekdays: 'Domingu_Segunda_Tersa_Kuarta_Kinta_Sesta_Sabadu'.split('_'),
diff --git a/esm/locale/tg.js b/esm/locale/tg.js
index 536df0b967beb89fed160c75a63b5ef8c270d246..c79331fe0f483d08abc8fb593f71b32a6ec7abc2 100644
--- a/esm/locale/tg.js
+++ b/esm/locale/tg.js
@@ -1,5 +1,5 @@
 // Tajik [tg]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tg',
   weekdays: 'якшанбе_душанбе_сешанбе_чоршанбе_панҷшанбе_ҷумъа_шанбе'.split('_'),
diff --git a/esm/locale/th.js b/esm/locale/th.js
index 5cbcdf2ad635c47c56e07e4fb01ec3044b2fbb0b..df3a3bc89ad2ec112f0a0478a2bde49297b5da31 100644
--- a/esm/locale/th.js
+++ b/esm/locale/th.js
@@ -1,5 +1,5 @@
 // Thai [th]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'th',
   weekdays: 'อาทิตย์_จันทร์_อังคาร_พุธ_พฤหัสบดี_ศุกร์_เสาร์'.split('_'),
diff --git a/esm/locale/tk.js b/esm/locale/tk.js
index 93390f18b857ccd3dd0e6f8430f6f43d87532312..d877e299491ba1b287bbaee6d11771f2c159b8b8 100644
--- a/esm/locale/tk.js
+++ b/esm/locale/tk.js
@@ -1,5 +1,5 @@
 // Turkmen [tk]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tk',
   weekdays: 'Ýekşenbe_Duşenbe_Sişenbe_Çarşenbe_Penşenbe_Anna_Şenbe'.split('_'),
diff --git a/esm/locale/tl-ph.js b/esm/locale/tl-ph.js
index 0fa84f3a7213157d2eecbc0fd61385b75f2ba64b..b73b29484068f5dc13f3e6adf059525a9e2c2f84 100644
--- a/esm/locale/tl-ph.js
+++ b/esm/locale/tl-ph.js
@@ -1,5 +1,5 @@
 // Tagalog (Philippines) [tl-ph]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tl-ph',
   weekdays: 'Linggo_Lunes_Martes_Miyerkules_Huwebes_Biyernes_Sabado'.split('_'),
diff --git a/esm/locale/tlh.js b/esm/locale/tlh.js
index 30f52fe74116fbc6963a7796ffa2488cc5a276f7..358c372bcdbdd0c50de27adb4b81896c866514f4 100644
--- a/esm/locale/tlh.js
+++ b/esm/locale/tlh.js
@@ -1,5 +1,5 @@
 // Klingon [tlh]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tlh',
   weekdays: 'lojmItjaj_DaSjaj_povjaj_ghItlhjaj_loghjaj_buqjaj_ghInjaj'.split('_'),
diff --git a/esm/locale/tr.js b/esm/locale/tr.js
index e7fe24fb3b1ef426dafa8390d4aab43072ea636b..977a8352cc19d2858ef44e36227e25b7aaa0fe29 100644
--- a/esm/locale/tr.js
+++ b/esm/locale/tr.js
@@ -1,5 +1,5 @@
 // Turkish [tr]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tr',
   weekdays: 'Pazar_Pazartesi_Salı_Çarşamba_Perşembe_Cuma_Cumartesi'.split('_'),
diff --git a/esm/locale/tzl.js b/esm/locale/tzl.js
index 9fa0cd2b97076f9e4d017c49e2c5a8c2450d4b77..ff98831db85d9377d5975fc71f68876b1afe7134 100644
--- a/esm/locale/tzl.js
+++ b/esm/locale/tzl.js
@@ -1,5 +1,5 @@
 // Talossan [tzl]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tzl',
   weekdays: 'Súladi_Lúneçi_Maitzi_Márcuri_Xhúadi_Viénerçi_Sáturi'.split('_'),
diff --git a/esm/locale/tzm-latn.js b/esm/locale/tzm-latn.js
index e5ac6aff76daa9151f783a887a134cb5025f63bd..da9178a73d139a28569b512a9c9e8f5cab504c5c 100644
--- a/esm/locale/tzm-latn.js
+++ b/esm/locale/tzm-latn.js
@@ -1,5 +1,5 @@
 // Central Atlas Tamazight Latin [tzm-latn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tzm-latn',
   weekdays: 'asamas_aynas_asinas_akras_akwas_asimwas_asiḍyas'.split('_'),
diff --git a/esm/locale/tzm.js b/esm/locale/tzm.js
index d94a6c09383257c49026aa9ef83fe2f6cf4946d8..3d5d6d5061cd7f025620a64b4b4063aac22548fe 100644
--- a/esm/locale/tzm.js
+++ b/esm/locale/tzm.js
@@ -1,5 +1,5 @@
 // Central Atlas Tamazight [tzm]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'tzm',
   weekdays: 'ⴰⵙⴰⵎⴰⵙ_ⴰⵢⵏⴰⵙ_ⴰⵙⵉⵏⴰⵙ_ⴰⴽⵔⴰⵙ_ⴰⴽⵡⴰⵙ_ⴰⵙⵉⵎⵡⴰⵙ_ⴰⵙⵉⴹⵢⴰⵙ'.split('_'),
diff --git a/esm/locale/ug-cn.js b/esm/locale/ug-cn.js
index d3d6392a822782c722f69aefe87f2af1ff43eea5..ee72c408902020bd646d32f5cbb25d7315d615c9 100644
--- a/esm/locale/ug-cn.js
+++ b/esm/locale/ug-cn.js
@@ -1,5 +1,5 @@
 // Uyghur (China) [ug-cn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ug-cn',
   weekdays: 'يەكشەنبە_دۈشەنبە_سەيشەنبە_چارشەنبە_پەيشەنبە_جۈمە_شەنبە'.split('_'),
diff --git a/esm/locale/uk.js b/esm/locale/uk.js
index 3c70b1329c5f6ec501c4568bfef4371abfcc1d7e..c1648a93affb72c8403cb126aad84fb2a82687e0 100644
--- a/esm/locale/uk.js
+++ b/esm/locale/uk.js
@@ -1,5 +1,5 @@
 // Ukrainian [uk]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var monthFormat = 'січня_лютого_березня_квітня_травня_червня_липня_серпня_вересня_жовтня_листопада_грудня'.split('_');
 var monthStandalone = 'січень_лютий_березень_квітень_травень_червень_липень_серпень_вересень_жовтень_листопад_грудень'.split('_');
 var MONTHS_IN_FORMAT = /D[oD]?(\[[^[\]]*\]|\s)+MMMM?/;
diff --git a/esm/locale/ur.js b/esm/locale/ur.js
index 7464c1ebc93c035cfc8b1131dd2f2857cf595572..3cc09791259eb4fcf3159a4b2f09ee8c8780a857 100644
--- a/esm/locale/ur.js
+++ b/esm/locale/ur.js
@@ -1,5 +1,5 @@
 // Urdu [ur]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'ur',
   weekdays: 'اتوار_پیر_منگل_بدھ_جمعرات_جمعہ_ہفتہ'.split('_'),
diff --git a/esm/locale/uz-latn.js b/esm/locale/uz-latn.js
index befdfee44fb78d14ca17fac0692a49cb19507082..787aad10ef8d1e4eb0adea7df098619949240b1d 100644
--- a/esm/locale/uz-latn.js
+++ b/esm/locale/uz-latn.js
@@ -1,5 +1,5 @@
 // Uzbek Latin [uz-latn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'uz-latn',
   weekdays: 'Yakshanba_Dushanba_Seshanba_Chorshanba_Payshanba_Juma_Shanba'.split('_'),
diff --git a/esm/locale/uz.js b/esm/locale/uz.js
index 443326373b1d6ebed3f533feb01221f067e1248c..0b461f1a919eaa605b472e759f6b8ec70f15c2d7 100644
--- a/esm/locale/uz.js
+++ b/esm/locale/uz.js
@@ -1,5 +1,5 @@
 // Uzbek [uz]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'uz',
   weekdays: 'Якшанба_Душанба_Сешанба_Чоршанба_Пайшанба_Жума_Шанба'.split('_'),
diff --git a/esm/locale/vi.js b/esm/locale/vi.js
index f55cc732b88c30d58310ee860cc050011dc7a037..eb859f4886aa03e0a9990bebb5d866fd407ca066 100644
--- a/esm/locale/vi.js
+++ b/esm/locale/vi.js
@@ -1,5 +1,5 @@
 // Vietnamese [vi]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'vi',
   weekdays: 'chủ nhật_thứ hai_thứ ba_thứ tư_thứ năm_thứ sáu_thứ bảy'.split('_'),
diff --git a/esm/locale/x-pseudo.js b/esm/locale/x-pseudo.js
index ceb67827d86501182b01caed4827bc85b243c40b..37273419a1a62081a802f3ae19f36fb8eee665e4 100644
--- a/esm/locale/x-pseudo.js
+++ b/esm/locale/x-pseudo.js
@@ -1,5 +1,5 @@
 // Pseudo [x-pseudo]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'x-pseudo',
   weekdays: 'S~úñdá~ý_Mó~ñdáý~_Túé~sdáý~_Wéd~ñésd~áý_T~húrs~dáý_~Fríd~áý_S~átúr~dáý'.split('_'),
diff --git a/esm/locale/yo.js b/esm/locale/yo.js
index 1f7946846d2ff659390444642b89f790cfe5354c..847438e53fa48aa37ca35a2f829b0bc2a1338dc5 100644
--- a/esm/locale/yo.js
+++ b/esm/locale/yo.js
@@ -1,5 +1,5 @@
 // Yoruba Nigeria [yo]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'yo',
   weekdays: 'Àìkú_Ajé_Ìsẹ́gun_Ọjọ́rú_Ọjọ́bọ_Ẹtì_Àbámẹ́ta'.split('_'),
diff --git a/esm/locale/zh-cn.js b/esm/locale/zh-cn.js
index 1a7ebf4bdf7e3eec8f56092f22e062ecba33b69b..02adaeea062e63fc594938ef017ed14775c10c10 100644
--- a/esm/locale/zh-cn.js
+++ b/esm/locale/zh-cn.js
@@ -1,5 +1,5 @@
 // Chinese (China) [zh-cn]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'zh-cn',
   weekdays: '星期日_星期一_星期二_星期三_星期四_星期五_星期六'.split('_'),
diff --git a/esm/locale/zh-hk.js b/esm/locale/zh-hk.js
index 8ada9a018dbef08bb6a55a86f7e23c4cac47fdf7..e31955acfa59ba3a85fbf5e21ad5d64fc0a35337 100644
--- a/esm/locale/zh-hk.js
+++ b/esm/locale/zh-hk.js
@@ -1,5 +1,5 @@
 // Chinese (Hong Kong) [zh-hk]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'zh-hk',
   months: '一月_二月_三月_四月_五月_六月_七月_八月_九月_十月_十一月_十二月'.split('_'),
diff --git a/esm/locale/zh-tw.js b/esm/locale/zh-tw.js
index ada89ee41ef62499ba501ba7dd7123baecbd47e2..4343b47a672818482c9314fe39d386ef7d8fb9cc 100644
--- a/esm/locale/zh-tw.js
+++ b/esm/locale/zh-tw.js
@@ -1,5 +1,5 @@
 // Chinese (Taiwan) [zh-tw]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'zh-tw',
   weekdays: '星期日_星期一_星期二_星期三_星期四_星期五_星期六'.split('_'),
diff --git a/esm/locale/zh.js b/esm/locale/zh.js
index b98ab70df06526bca8bc8401e532444819e74d61..75848dd3793e71abe479ee31784c01959f62e042 100644
--- a/esm/locale/zh.js
+++ b/esm/locale/zh.js
@@ -1,5 +1,5 @@
 // Chinese [zh]
-import dayjs from '../index';
+import dayjs from '../index.js';
 var locale = {
   name: 'zh',
   weekdays: '星期日_星期一_星期二_星期三_星期四_星期五_星期六'.split('_'),
diff --git a/esm/plugin/advancedFormat/index.js b/esm/plugin/advancedFormat/index.js
index f45e4e0042159eb361a92b9cdf3b8ada98ec9181..9cebcdecd2d7b477bd7c09a0ffb5c6fab3a1c8f1 100644
--- a/esm/plugin/advancedFormat/index.js
+++ b/esm/plugin/advancedFormat/index.js
@@ -1,4 +1,4 @@
-import { FORMAT_DEFAULT } from '../../constant';
+import { FORMAT_DEFAULT } from '../../constant.js';
 export default (function (o, c) {
   // locale needed later
   var proto = c.prototype;
diff --git a/esm/plugin/buddhistEra/index.js b/esm/plugin/buddhistEra/index.js
index 76ce44cca926d87f93e6d1729635b6499a18517d..fc2d58dc911ae17fd1077a15f76b935dc1eb88b3 100644
--- a/esm/plugin/buddhistEra/index.js
+++ b/esm/plugin/buddhistEra/index.js
@@ -1,4 +1,4 @@
-import { FORMAT_DEFAULT } from '../../constant';
+import { FORMAT_DEFAULT } from '../../constant.js';
 export default (function (o, c) {
   // locale needed later
   var proto = c.prototype;
diff --git a/esm/plugin/customParseFormat/index.js b/esm/plugin/customParseFormat/index.js
index b5363f2a88add165f895d70dba9ccdaa1bafd3df..b4838d4cca25265994c4a6047dbb854a6fc88377 100644
--- a/esm/plugin/customParseFormat/index.js
+++ b/esm/plugin/customParseFormat/index.js
@@ -1,4 +1,4 @@
-import { u } from '../localizedFormat/utils';
+import { u } from '../localizedFormat/utils.js';
 var formattingTokens = /(\[[^[]*\])|([-_:/.,()\s]+)|(A|a|Q|YYYY|YY?|ww?|MM?M?M?|Do|DD?|hh?|HH?|mm?|ss?|S{1,3}|z|ZZ?)/g;
 var match1 = /\d/; // 0 - 9
 
diff --git a/esm/plugin/duration/index.js b/esm/plugin/duration/index.js
index a241d4b202e99c61467639a5756c586e0e50ceb7..ee2cdd109ceadda80705b369c3d15789c4c9c4a0 100644
--- a/esm/plugin/duration/index.js
+++ b/esm/plugin/duration/index.js
@@ -1,4 +1,4 @@
-import { MILLISECONDS_A_DAY, MILLISECONDS_A_HOUR, MILLISECONDS_A_MINUTE, MILLISECONDS_A_SECOND, MILLISECONDS_A_WEEK, REGEX_FORMAT } from '../../constant';
+import { MILLISECONDS_A_DAY, MILLISECONDS_A_HOUR, MILLISECONDS_A_MINUTE, MILLISECONDS_A_SECOND, MILLISECONDS_A_WEEK, REGEX_FORMAT } from '../../constant.js';
 var MILLISECONDS_A_YEAR = MILLISECONDS_A_DAY * 365;
 var MILLISECONDS_A_MONTH = MILLISECONDS_A_YEAR / 12;
 var durationRegex = /^(-|\+)?P(?:([-+]?[0-9,.]*)Y)?(?:([-+]?[0-9,.]*)M)?(?:([-+]?[0-9,.]*)W)?(?:([-+]?[0-9,.]*)D)?(?:T(?:([-+]?[0-9,.]*)H)?(?:([-+]?[0-9,.]*)M)?(?:([-+]?[0-9,.]*)S)?)?$/;
diff --git a/esm/plugin/isoWeek/index.js b/esm/plugin/isoWeek/index.js
index 289ea7ca1e4692fb45271d340365cf1dca8b6e52..fd8cd4c7890ec8e6d6d151dd048f4e47e1774cc9 100644
--- a/esm/plugin/isoWeek/index.js
+++ b/esm/plugin/isoWeek/index.js
@@ -1,4 +1,4 @@
-import { D, W, Y } from '../../constant';
+import { D, W, Y } from '../../constant.js';
 var isoWeekPrettyUnit = 'isoweek';
 export default (function (o, c, d) {
   var getYearFirstThursday = function getYearFirstThursday(year, isUtc) {
diff --git a/esm/plugin/localeData/index.js b/esm/plugin/localeData/index.js
index c48d92cbc6f8d0da3b7688f462727e911e010b51..ff00bac9c98499633376094f631bd5e7df773309 100644
--- a/esm/plugin/localeData/index.js
+++ b/esm/plugin/localeData/index.js
@@ -1,4 +1,4 @@
-import { t } from '../localizedFormat/utils';
+import { t } from '../localizedFormat/utils.js';
 export default (function (o, c, dayjs) {
   // locale needed later
   var proto = c.prototype;
diff --git a/esm/plugin/localizedFormat/index.js b/esm/plugin/localizedFormat/index.js
index 9defb1f4124f27bd6eda83a6112a95c2b205bd49..1c29b99373fb524631d28f35a838db2f5061bec5 100644
--- a/esm/plugin/localizedFormat/index.js
+++ b/esm/plugin/localizedFormat/index.js
@@ -1,5 +1,5 @@
-import { FORMAT_DEFAULT } from '../../constant';
-import { u, englishFormats } from './utils';
+import { FORMAT_DEFAULT } from '../../constant.js';
+import { u, englishFormats } from './utils.js';
 export default (function (o, c, d) {
   var proto = c.prototype;
   var oldFormat = proto.format;
diff --git a/esm/plugin/quarterOfYear/index.js b/esm/plugin/quarterOfYear/index.js
index e376889e5904ca65e408e58d98c860d27b2f3ecc..23f0daf39cbed6db28348f4c8d6c7011b5d0b07e 100644
--- a/esm/plugin/quarterOfYear/index.js
+++ b/esm/plugin/quarterOfYear/index.js
@@ -1,4 +1,4 @@
-import { Q, M, D } from '../../constant';
+import { Q, M, D } from '../../constant.js';
 export default (function (o, c) {
   var proto = c.prototype;
 
diff --git a/esm/plugin/relativeTime/index.js b/esm/plugin/relativeTime/index.js
index 88fdbbe833470cc7c76dad3b62ca8f7befc27091..b1c64681a43487ea72d03630543a8140a4cd15da 100644
--- a/esm/plugin/relativeTime/index.js
+++ b/esm/plugin/relativeTime/index.js
@@ -1,4 +1,4 @@
-import * as C from '../../constant';
+import * as C from '../../constant.js';
 export default (function (o, c, d) {
   o = o || {};
   var proto = c.prototype;
diff --git a/esm/plugin/timezone/index.js b/esm/plugin/timezone/index.js
index 2f63de26b223b2c9c6214ecefbfec748e6c43fda..5f0e435bbfc746028efe4e1345d4910aff7eba27 100644
--- a/esm/plugin/timezone/index.js
+++ b/esm/plugin/timezone/index.js
@@ -1,4 +1,4 @@
-import { MIN, MS } from '../../constant';
+import { MIN, MS } from '../../constant.js';
 var typeToPos = {
   year: 0,
   month: 1,
diff --git a/esm/plugin/utc/index.js b/esm/plugin/utc/index.js
index a8a05f56c405466931d6ffec9202b464c3f0f50f..552551e61fc268ff46cfbeb38963364cbfbf4408 100644
--- a/esm/plugin/utc/index.js
+++ b/esm/plugin/utc/index.js
@@ -1,4 +1,4 @@
-import { MILLISECONDS_A_MINUTE, MIN } from '../../constant';
+import { MILLISECONDS_A_MINUTE, MIN } from '../../constant.js';
 var REGEX_VALID_OFFSET_FORMAT = /[+-]\d\d(?::?\d\d)?/g;
 var REGEX_OFFSET_HOURS_MINUTES_FORMAT = /([+-]|\d\d)/g;
 
diff --git a/esm/plugin/weekOfYear/index.js b/esm/plugin/weekOfYear/index.js
index c92406eb628aaaf27183b2b2adf6355f09694a7e..910e96ebc2fb73bdc5cdd92fbe43e072267ad974 100644
--- a/esm/plugin/weekOfYear/index.js
+++ b/esm/plugin/weekOfYear/index.js
@@ -1,4 +1,4 @@
-import { MS, Y, D, W } from '../../constant';
+import { MS, Y, D, W } from '../../constant.js';
 export default (function (o, c, d) {
   var proto = c.prototype;
 
diff --git a/esm/utils.js b/esm/utils.js
index b5a81312b702adf2f60a28db5bee15c4e8e6828b..7df153ad06bacb6746b3f11908bb3f41320489a1 100644
--- a/esm/utils.js
+++ b/esm/utils.js
@@ -1,4 +1,4 @@
-import * as C from './constant';
+import * as C from './constant.js';
 
 var padStart = function padStart(string, length, pad) {
   var s = String(string);
