diff --git a/fix-esm-import-path.js b/fix-esm-import-path.js
index 2c3a802a5b897d88db056192b5991fde21501d6b..1817a82b10908733c6046b8914d9f7f2495c1f39 100755
--- a/fix-esm-import-path.js
+++ b/fix-esm-import-path.js
@@ -162,7 +162,7 @@ function scanImport({ srcFile, importCode, name }) {
           srcFile,
           importCode,
           from: name,
-          to: importFile.startsWith(importName + '/index')
+          to: importFile.startsWith(importName + `${path.sep}index`)
             ? name + '/index.js'
             : name + '.js',
         })
