diff --git a/fix-esm-import-path.js b/fix-esm-import-path.js
index 2c3a802..1817a82 100644
--- a/fix-esm-import-path.js
+++ b/fix-esm-import-path.js
@@ -162,7 +162,7 @@ function scanImport({ srcFile, importCode, name }) {
           srcFile,
           importCode,
           from: name,
-          to: importFile.startsWith(importName + '/index')
+          to: importFile.startsWith(importName + `${path.sep}index`)
             ? name + '/index.js'
             : name + '.js',
         })
