diff --git a/build/index.js b/build/index.js
index 2e6c16c33c2f588cf8d6a5717ac776bb8908bd0d..0baf969a54c1b3d9c1104ec8d9fee68523cbe348 100644
--- a/build/index.js
+++ b/build/index.js
@@ -81,6 +81,10 @@ class JSDOMEnvironment {
       }
     );
     const global = (this.global = this.dom.window);
+    global.fetch = fetch;
+    global.Headers = Headers;
+    global.Request = Request;
+    global.Response = Response;
     if (global == null) {
       throw new Error('JSDOM did not return a Window object');
     }
