diff --git a/node_modules/tailwindcss/lib/lib/expandTailwindAtRules.js b/node_modules/tailwindcss/lib/lib/expandTailwindAtRules.js
index 338a619..98aa7d2 100644
--- a/node_modules/tailwindcss/lib/lib/expandTailwindAtRules.js
+++ b/node_modules/tailwindcss/lib/lib/expandTailwindAtRules.js
@@ -166,7 +166,7 @@ function expandTailwindAtRules(context) {
                 }
             ]);
         }
-        const BATCH_SIZE = 500;
+        const BATCH_SIZE = process.platform === 'win32' ? 1 : 500;
         for(let i = 0; i < regexParserContent.length; i += BATCH_SIZE){
             let batch = regexParserContent.slice(i, i + BATCH_SIZE);
             await Promise.all(batch.map(async ([{ file , content  }, { transformer , extractor  }])=>{
