diff --git a/node_modules/tailwindcss/lib/lib/expandTailwindAtRules.js b/node_modules/tailwindcss/lib/lib/expandTailwindAtRules.js
index 43c3240..212bfc1 100644
--- a/node_modules/tailwindcss/lib/lib/expandTailwindAtRules.js
+++ b/node_modules/tailwindcss/lib/lib/expandTailwindAtRules.js
@@ -165,12 +165,21 @@ function expandTailwindAtRules(context) {
         //   getClassCandidatesOxide(file, transformer(content), extractor, candidates, seen)
         // }
         } else {
-            await Promise.all(context.changedContent.map(async ({ file , content , extension  })=>{
-                let transformer = getTransformer(context.tailwindConfig, extension);
-                let extractor = getExtractor(context, extension);
-                content = file ? await _fs.default.promises.readFile(file, "utf8") : content;
-                getClassCandidates(transformer(content), extractor, candidates, seen);
-            }));
+            if(process.platform === 'win32') {
+                for (let { file, content, extension } of context.changedContent) {
+                    let transformer = getTransformer(context.tailwindConfig, extension)
+                    let extractor = getExtractor(context, extension)
+                    content = file ? _fs.default.readFileSync(file, 'utf8') : content
+                    getClassCandidates(transformer(content), extractor, candidates, seen)
+                }
+            } else {
+                await Promise.all(context.changedContent.map(async ({ file , content , extension  })=>{
+                    let transformer = getTransformer(context.tailwindConfig, extension);
+                    let extractor = getExtractor(context, extension);
+                    content = file ? await _fs.default.promises.readFile(file, "utf8") : content;
+                    getClassCandidates(transformer(content), extractor, candidates, seen);
+                }));
+            }
         }
         env.DEBUG && console.timeEnd("Reading changed files");
         // ---
